frontend/src/pages/NewMetadataType/NewMetadataTypeBody.jsx

```diff
@@ -70,24 +71,31 @@ export const NewMetadataTypeBody = () => {
   const allTags = useGetTags();
   const [name, setName] = useState('');
   const [description, setDescription] = useState('');
+  const [categoryUuid, setCategoryUuid] = useState('');
   const [tags, setTags] = useState([]);
   const [submissionStatus, setSubmissionStatus] = useState('');
   const [loading, setLoading] = useState(false);
 
   const dispatch = useDispatch();
   const userSelector = useSelector((state) => state.user);
 
+  const onCategoryChange = (e) => setCategoryUuid(e.currentTarget.value);
+
   const submit = async (e) => {
     e.preventDefault();
+    if (categoryUuid === '') {
```

Comment:
.isEmpty() or null ?

Reply:
Might be simplified to `if (!categoryUuid)´ if I'm not mistaken.

Reply:
It can't be `null` since the default is `''`, if that was the question here